resources/views/controller.blade.php

```diff
@@ -200,15 +227,21 @@ public function update(Update{{ $modelBaseName }} $request, {{ $modelBaseName }}
 @if (count($relations['belongsToMany']))
 @foreach($relations['belongsToMany'] as $belongsToMany)
         // But we do have a {{ $belongsToMany['related_table'] }}, so we need to attach the {{ $belongsToMany['related_table'] }} to the {{ $modelVariableName }}
-        if($request->has('{{ $belongsToMany['related_table'] }}')) {
+        if ($request->has('{{ $belongsToMany['related_table'] }}')) {
             ${{ $modelVariableName }}->{{ $belongsToMany['related_table'] }}()->sync(collect($request->input('{{ $belongsToMany['related_table'] }}', []))->map->id->toArray());
         }
 @endforeach
 
 @endif
 @endif
         if ($request->ajax()) {
-            return ['redirect' => url('admin/{{ $resource }}'), 'message' => trans('brackets/admin-ui::admin.operation.succeeded')];
+            return [
+                'redirect' => url('admin/{{ $resource }}'),
+                'message' => trans('brackets/admin-ui::admin.operation.succeeded'),
+@if($containsPublishedAtColumn)
```

Comment:
Why only with published at we send an object?

Reply:
Because when you publish from listing we need to change publish_at column in listing without refresh page, this functionality is also implemented in craftable-frontend